OJP/OJP_JourneySupport.xsd

```diff
@@ -93,6 +93,11 @@
 				</xs:annotation>
 			</xs:element>
 			<xs:element ref="OperatorRef" minOccurs="0"/>
+			<xs:element name="additionalOperatorRef" type="siri:OperatorRefStructure" minOccurs="0" maxOccurs="unbounded">
```

Comment:
I think NeTEx takes an approach where additionalOperators would be a list, with ref elements. Can this be checked?

Reply:
I guess it would. But we didn't in all occasions. Can start here, if the group wants it.

Reply:
Changed